cmd/root.go

```diff
@@ -0,0 +1,129 @@
+package cmd
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+
+	"github.com/gin-gonic/gin"
+	"github.com/spf13/cobra"
+	"github.com/subosito/gotenv"
+
+	"github.com/codefluence-x/altair/core"
+	"github.com/codefluence-x/altair/loader"
+	"github.com/codefluence-x/journal"
+)
+
+var (
+	dbConfigs    map[string]core.DatabaseConfig = map[string]core.DatabaseConfig{}
+	databases    map[string]*sql.DB             = map[string]*sql.DB{}
+	appConfig    core.AppConfig
+	pluginBearer core.PluginBearer
+	apiEngine    *gin.Engine
+)
```

Comment:
Could all of this variable is injected from main package, so we can create a test for each function of `cmd` package and keep the coverage on point?